ts/parts-gantt/Tree.ts

```diff
@@ -83,7 +83,7 @@ var isFunction = function (x: unknown): x is Function {
  * @function Highcharts.Tree#getListOfParents
  *
  * @param {Array<*>} data
- *        List of points set in options. `Array<*>.parent`is parent id of point.
```

Comment:
the same question as above :)

Reply:
No it won't affect the structure of the tree. This is just the description.